Makefile

```diff
@@ -24,7 +24,7 @@ CORE_VDE_PREFIX ?= $(OUTDIR)/dependencies/vde/opt/finch
 LICENSEDIR := $(OUTDIR)/license-files
 VERSION := $(shell git describe --match 'v[0-9]*' --dirty='.modified' --always --tags)
 GITCOMMIT := $(shell git rev-parse HEAD)$(shell test -z "$(git status --porcelain)" || echo .m)
-LDFLAGS := "-X $(PACKAGE)/pkg/version.Version=$(VERSION) -X $(PACKAGE)/pkg/version.GitCommit=$(GITCOMMIT)"
+LDFLAGS = "-w -X $(PACKAGE)/pkg/version.Version=$(VERSION) -X $(PACKAGE)/pkg/version.GitCommit=$(GITCOMMIT)"
```

Comment:
Curious if this reduces the size of the finch binary?
Also, what are my options if I need a debug build?

Reply:
Yes, it did reduce the executable size by a few megabytes. According to https://github.com/golang/go/issues/62577, Embedding debug symbols like this is wrong on macOS anyway. We can probably add `-w` only on macOS if we need the debugging symbols on other platforms. The Makefile is only really used for release builds right now though.